requirements.txt

```diff
@@ -1,4 +1,4 @@
-py-cord[speed]==2.4.1
+py-cord==2.5.0
 python-dotenv==0.21.1
-mcstatus==11.0.1
-mysql-connector-python==8.0.32
+mcstatus==11.1.1
+mysql-connector-python==8.3.0
```

Comment:
Use >= for these

Reply:
Not ~= for compatibility equality?

Reply:
~= will allow any greater subversion, but no major version update. That way updates never break existing code ideally. >= allows e.g. pycord 3.0 to be used too, which has breaking changes as a major update

Reply:
No.

Reply:
I'm aware that breaking changes _could_ break it, but principly, when a package is updated to that point it should be updated here as well anyways. This'll both keep us on the ropes for that and make sure we're up to date with anything, even if major releases don't necessarily have breaking changes that can interfere with our code - which also happens often enough.